CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 
 ### Bugfixes
-
+- [#994](https://github.com/openscope/openscope/issues/994) - fix "Aircraft vectored off ILS maintain over-precise altitude"
```

Comment:
@felixscheffer try to keep changelog entries as added lines instead of edited lines, makes things easier at merge time.